### PR TITLE
Add --junittime=call option

### PR DESCRIPTION
**Pull request by aparamon**
Implements https://github.com/pytest-dev/pytest/issues/4483
Please review and let me know if docs/changelog update is required for this simple change.
attn @MuchkoM

### CODE REVIEW

File: src/_pytest/junitxml.py [hunk 1]
```diff
@@ -361,12 +371,15 @@ def mangle_test_address(address):
 
 
 class LogXML(object):
-    def __init__(self, logfile, prefix, suite_name="pytest", logging="no"):
+    def __init__(
+        self, logfile, prefix, suite_name="pytest", logging="no", report_duration=None
```
Comment (nicoddemus) [inline src/_pytest/junitxml.py]: To make things simpler I propose to use `report_duration="total"`; this way you can remove the `not self.report_duration` from the check in `update_testcase_duration`

**Comment by nicoddemus** [comment]
> this should be a ini option

I was wondering the same thing

File: src/_pytest/junitxml.py (hunk 1 above)
Comment (aparamon) [reply src/_pytest/junitxml.py]: That was the initial version, but then I thought that duplicating const "total" at https://github.com/aparamon/pytest/blob/cfbd387a5d7a27a3e7dfb754d7cd4482e931822b/testing/test_junitxml.py#L748
would be less desirable. (Also overall, using `None` as default value looks good interface.)

**Comment by aparamon** [comment]
Thanks for the review @nicoddemus, @RonnyPfannschmidt!
I'll try harder to install `pre-commit` -- for some reason it keeps failing on my box.
Will switch to ini option and update the docs/changelog/authors shortly.

**Pull request merged**